### ISSUE TITLE
How much has browser list outdated?

### ISSUE THREAD

**Issue by nerkn** [browserslist/update-db#46]
I got this error while compiling nextjs project, which is fairly new (1 month old) . Documents mention "**project 2 years ago**" . It would better to print :
```
Browserslist: caniuse-lite is 15552002 seconds old. Time to run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme

```

Orginal message :
```
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
```

**Comment by ai** [comment]
Good idea!

**Comment by SergeyKrasnolobov** [comment]
@ai https://github.com/browserslist/browserslist/pull/848

**Comment by mezhnin** [comment]
@ai https://github.com/browserslist/browserslist/pull/849

**Comment by ai** [comment]
Fixed https://github.com/browserslist/browserslist/pull/849

@mezhnin was the first with clean solution (tests, pluralization, reusing time calc)

**Comment by ai** [comment]
Released in Browserslist [4.24.1](https://github.com/browserslist/browserslist/releases/tag/4.24.1)

**Comment by ai** [comment]
I also made warning text more clear in 4.24.2:

> Browserslist: browsers data (caniuse-lite) is 20 months old.

Now it explain why you should update it (browsers data).
